packages/core/src/ruleset/meta/rule.schema.json

```diff
@@ -72,11 +72,6 @@
         "else": {
           "$ref": "#/$defs/Then"
         }
-      },
-      "type": {
```

Comment:
I'd say keep it, otherwise it's a breaking change if there are folks that using it - the ruleset validation will probably mark it as a non-recognized (additional) property due to https://github.com/stoplightio/spectral/blob/813704d3f57a392a7083542b5df453d789a19e0b/packages/core/src/ruleset/meta/rule.schema.json#L78